ansible_events_ui/api/activation.py

```diff
@@ -1,8 +1,125 @@
 """Activation API endpoints."""
-from fastapi import APIRouter
+import logging
+import json
+
+from fastapi import APIRouter, Depends
+import asyncio
+
+import sqlalchemy.orm
+from sqlalchemy import insert, select
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from ansible_events_ui.db.dependency import (
+    get_db_session,
+    get_db_session_factory,
+)
+from ansible_events_ui.db.models import (
```

Comment:
```suggestion
from ansible_events_ui.db import models
```

Minor: Better to use module import for models too.
When tables are changed into declarative models, they become classes and naming may clash with schemas module.